Classes/Console/Command/Upgrade/UpgradeAllCommand.php

```diff
@@ -40,7 +40,7 @@ protected function createNativeDefinition(): array
             new InputOption(
                 'arguments',
                 'a',
-                InputOption::VALUE_REQUIRED,
+                InputOption::VALUE_REQUIRED | InputOption::VALUE_IS_ARRAY,
```

Comment:
while this fixes the issue, it introduces a change in behviour which isn't itentional